src/components/Clock.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react';
+
+type State = {
+  today: Date
+};
+
+type Props = {
+  name: string,
+};
+
+export class Clock extends React.Component<Props, State> {
+  state = {
+    today: new Date(),
+  };
+
+  timerId: number | undefined;
+
+  componentDidMount() {
+    this.timerId = window.setInterval(() => {
+      this.setState({ today: new Date() });
+
+      // eslint-disable-next-line no-console
+      console.info(this.state.today.toUTCString().slice(-12, -4));
```

Comment:
Better to move this logic to function `.toUTCString().slice(-12, -4)` so you will be able to reuse it